core/templates/pages/exploration-editor-page/translation-tab/modal-templates/welcome-translation-modal.component.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the translation tab.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { ContextService } from 'services/context.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
+
+@Component({
+  selector: 'oppia-welcome-translation-modal',
+  templateUrl: './welcome-translation-modal.component.html'
+})
+
+export class WelcomeTranslationModalComponent
+  extends ConfirmOrCancelModal implements OnInit {
+  explorationId: string;
+  translationWelcomeImgUrl: string;
+
+  constructor(
+    private ngbActiveModal: NgbActiveModal,
+    private urlInterpolationService: UrlInterpolationService,
+    private contextService: ContextService,
+    private siteAnalyticsService: SiteAnalyticsService,
+  ) {
+    super(ngbActiveModal);
+  }
+
+  ngOnInit(): void {
+    this.explorationId = this.contextService.getExplorationId();
+    this.siteAnalyticsService
+      .registerTutorialModalOpenEvent(this.explorationId);
+    this.translationWelcomeImgUrl = this.urlInterpolationService
+      .getStaticImageUrl('/general/editor_welcome.svg');
```

Comment:
Let's prefer to not break up the function calls. Can `this.siteAnalyticsService.registerTutorialModalOpenEvent(` fit on one line? And also `this.urlInterpolationService.getStaticImageUrl(`?

Reply:
after doing the function call in single line there was an 83 word cross eslint error